src/runtime/runtime_atsamd21.go

```diff
@@ -7,6 +7,7 @@ import (
 	"device/arm"
 	"device/sam"
 	"machine"
+	"reflect"
```

Comment:
I do not think we want to require the "reflect" package here, it has to increase the minimum memory footprint, no?

Reply:
Using reflect makes a big difference in ROM size, doesn't it?
I hadn't noticed that.

wioterminal:
| - | flash (byte) | ram (byte) |
| -- | --: | --: |
| dev (4b2edc9a26a84b4870d61ddf09f70369ab9f6f1e) | 5260 | 4276 |
| #2631 (reflect) | 13000 | 6500 |
| #2631 (w/o reflect) | 8160 | 6504 |

Reply:
I fixed it with a new commit.
However, I'm wondering what I should do about @bgould's point.